DS_Assignment_6/Employee/Employee.java

```diff
@@ -0,0 +1,43 @@
+package Employee;
+
+public class Employee implements Comparable<Employee> {
+
+	private int emp_ID;
+	private String name;
+	private String address;
+
+	public Employee(int emp_ID, String name, String address) {
+		this.emp_ID = emp_ID;
+		this.name = name;
+		this.address = address;
+	}
+
+	public int getEmpID() {
+		return this.emp_ID;
+	}
+
+	public String getName() {
+		return this.name;
+	}
+
+	public String getAddress() {
+		return this.address;
+	}
+
+
+	/**
+	 * Overriding compareTo method and 
+	 * Comparing according to employee IDs(Natural Order)
+	 */
+	@Override
+	public int compareTo(Employee o) {
+		return this.getEmpID() - o.getEmpID();

```

Comment:
As part of session i mention that it is best to use campareTo instead of this. So in give case we can do this.getEmpID().compareTo(o.getEmpID());